drivers/power/reset/gpio-poweroff.c

```diff
@@ -76,14 +80,21 @@ static int gpio_poweroff_probe(struct platform_device *pdev)
 	if (IS_ERR(reset_gpio))
 		return PTR_ERR(reset_gpio);
 
+	export = of_property_read_bool(pdev->dev.of_node, "export");
```

Comment:
This block appears to come from [this commit](https://github.com/raspberrypi/linux/commit/fbaddd65c8e5d588ffee8dc8b5b7b9faa8434387). However, you missed the [variable declaration](https://github.com/raspberrypi/linux/commit/fbaddd65c8e5d588ffee8dc8b5b7b9faa8434387#diff-ac6fa903566521bf39bc66f8de1995a3ad1b9faec23be1b459cad52997464e61R34). I double this commit can build now.

Reply:
You're right. Sorry about that.

Reply:
Added a commit that fixes this.